Makefile

```diff
@@ -56,12 +56,6 @@ export _NIX_KEEP ?= TMPDIR,BUILD_ENV,\
 # Useful for Android release builds
 TMP_BUILD_NUMBER := $(shell ./scripts/version/gen_build_no.sh | cut -c1-10)
 
-# MacOS root is read-only, read nix/README.md for details
```

Comment:
Do we have to update this `README.md` as well?

Reply:
Okay there is nothing about that warning in there so seems fine

Reply:
I see no reference in the readme about this.